Configuration/TCA/tx_generic_gallery_pictures.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+return [
+    'ctrl' => array(
+        'title' => 'LLL:EXT:generic_gallery/Resources/Private/Language/locallang_db.xlf:tx_genericgallery_domain_model_galleryitem',
+        'label' => 'title',
+        'tstamp' => 'tstamp',
+        'crdate' => 'crdate',
+        'cruser_id' => 'cruser_id',
+        'dividers2tabs' => TRUE,
+        'delete' => 'deleted',
+        'enablecolumns' => array(
+            'disabled' => 'hidden',
+            'starttime' => 'starttime',
+            'endtime' => 'endtime',
+        ),
+        'searchFields' => 'title,link,image,text_items,',
+        'typeicon_classes' => [
+            'default' => 'extensions-generic-gallery-pictures',
+        ],
+        'interface' => array(
```

Comment:
Gna... now I know why a closing ) was missing at the bottom, when I created this file... Thanks.